### ISSUE TITLE
new feature request

### ISSUE THREAD

**Issue by TWME-TW** [einTosti/BuildSystem#40]
1.Support DriveBackupV2 ( https://www.spigotmc.org/resources/drivebackupv2.79519/ )

    The world created by BuildSystem will automatically added to the backup list of DriveBackupV2, and maybe you can add command(GUI?) to create a backup (upload) to Google drive so that members can download their own world instantly.

2.Add a limit on the number of private worlds
Like `buildsystem.createworld.number`
`buildsystem.createprivate.number`

3.Add global limit on the number of worlds

4.Add "World prefix" 
   The world created by members will have a prefix, like `member-`, so that the name of the world (folder) they create becomes `member-XXXX`

(English is not my language, so there may be errors, please forgive me

**Comment by einTosti** [comment]
1.
> The world created by BuildSystem will automatically added to the backup list of DriveBackupV2, and maybe you can add command(GUI?) to create a backup (upload) to Google drive so that members can download their own world instantly.

I checked out the DriveBackupV2 API. Sadly it only allows me to start a backup, not to add worlds to the list

2.
> Add a limit on the number of private worlds
Like buildsystem.createworld.number
buildsystem.createprivate.number

You you mean as a permission? So that a player with the permission `buildsystem.createprivate.max.X` can only create max X worlds?
Or as a config option which applies globally

3.
> Add global limit on the number of worlds

See 2. 
(Config vs permission, global = „normal“ world?)

4.
> The world created by members will have a prefix, like member-, so that the name of the world (folder) they create becomes member-XXXX

How should this be setup? Via the config? 
Os it permission-based?

**Comment by TWME-TW** [comment]
1.
> 我檢查了 DriveBackupV2 API。可悲的是，它只允許我開始備份，不能將世界添加到列表中

OK, thanks

2.
> You you mean as a permission? So that a player with the permission `buildsystem.createprivate.max.X` can only create max X worlds?

I think it's based on permissions

4.
> How should this be setup? Via the config?Os it permission-based?

Via the config

The same concept as the original world name `world, world_end`

**Comment by TWME-TW** [comment]
I think World prefix has lower priority

`Limit amount of worlds` should be completed first.

Maybe there are permissions and config, one is global and the other is personal (of course, global restrictions have a higher priority than personal ones)

**Comment by TWME-TW** [comment]
An example:
`
WorldPrefix: 'member-' 
NewWorldName : '%player_id%'
`
Create New World ----> member-TWME

Change the world name to 'TEST' ---->  member-TEST

**Comment by TWME-TW** [comment]
Unimportant functions, already solved
